samples/panic2.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	n := panicTest()
+	fmt.Printf("panicTest: %d\n", n)
+}
+
+func panicTest() int {
+
+	var n int
+
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println(r)
+		}
+	}()
+
+	n = 2
```

Comment:
I wouldn't bother with these assignments, keep the examples as simple as possible.